css/pages/home.styl

```diff
@@ -118,40 +117,42 @@
         text-align: center
         line-height: 20px
 
-        img {
-          margin-top: 19px
+        vertical-center() {
+          position: relative
+          top: 50%
+          -webkit-transform: translateY(-50%);
+          -ms-transform: translateY(-50%);
```

Comment:
Are these still necessary? I think all the browsers in common use nowadays support unprefixed transform.

Reply:
[w3schools says they aren't](https://www.w3schools.com/cssref/css3_pr_transform.asp), so I'll remove them.